src/modules/roc_netio/target_uv/roc_netio/basic_port.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_uv/roc_netio/basic_port.h
+//! @brief basic network port.
+
+#ifndef ROC_NETIO_BASIC_PORT_H_
+#define ROC_NETIO_BASIC_PORT_H_
+
+#include "roc_core/iallocator.h"
+#include "roc_core/list_node.h"
+#include "roc_core/refcnt.h"
+
+#include "roc_packet/address.h"
```

Comment:
We usually use a single block (without newlines) for all includes for src/modules. We use separate blocks for external includes, src/modules includes, and src/tests includes.

Reply:
Fixed